.github/workflows/acceptance.yml

```diff
@@ -73,18 +81,35 @@ jobs:
       - name: Create extra resources
         run: |
           cmk -u $CLOUDSTACK_API_URL -k $CLOUDSTACK_API_KEY -s $CLOUDSTACK_SECRET_KEY -o json create project name=terraform displaytext=terraform
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: ${{ matrix.terraform-version }}
+          terraform_wrapper: false
       - name: Run acceptance test
         run: |
           make testacc
     services:
       cloudstack-simulator:
-        image: apache/cloudstack-simulator:${{ matrix.cloudstack_version }}
+        image: apache/cloudstack-simulator:${{ matrix.cloudstack-version }}
         ports:
           - 8080:5050
     strategy:
       fail-fast: false
       matrix:
-        cloudstack_version:
+        terraform-version:
+          - '0.12.*'
+          - '0.13.*'
+          - '0.14.*'
+          - '0.15.*'
+          - '1.0.*'
+          - '1.1.*'
+          - '1.2.*'
+          - '1.3.*'
+          - '1.4.*'
+          - '1.5.*'
+          - '1.6.*'
+          - '1.7.*'
```

Comment:
Does it make sense to test all the versions of terraform or only the non EOL?

Reply:
only non EOL.  I'd like to help/see more iterations pushed out vs. supporting EOL code.

Reply:
good catch @CodeBleu, I've added it now